oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/client/RegisteredClientRepository.java

```diff
@@ -47,4 +47,6 @@
 	@Nullable
 	RegisteredClient findByClientId(String clientId);
 
+	RegisteredClient saveClient(RegisteredClient registeredClient);
```

Comment:
Change to `void save(RegisteredClient registeredClient)`

Add javadoc

Reply:
Fixed